src/types/xlsx.ts

```diff
@@ -29,6 +30,7 @@ import { Alias, ExcelChartDefinition, Format, PaneDivision } from ".";
  *  - fills (XLSXFill): §18.8.20 (fill)
  *  - figure (XLSXFigure): §20.5.2.35 (wsDr (Worksheet Drawing))
  *  - fonts (XLSXFont): §18.8.22 (font)
+ *  - images (XLSXImageFile): §20.2.2.5 (pic)
```

Comment:
not that of an issue but this line belongs to PR https://github.com/odoo/o-spreadsheet/pull/1956

Reply:
note*